src/react/header/Header.tsx

```diff
@@ -0,0 +1,145 @@
+import clsx from 'clsx'
+import type { ReactNode } from 'react'
+import { useEffect, useRef } from 'react'
+import { MODlogo } from './MODlogo'
+
+interface NavigationOptions {
+  key?: string
+  content?: ReactNode | string
```

Comment:
The nunjunks api uses html or text. I think it's worth combining them into one and type check it. 
Also, it allows users to use `<Link/>` from Next.JS or Remix